### PR TITLE
Feature/dao

### PR DESCRIPTION
**Pull request by rmw1075**
merge finished DAO into master

### CODE REVIEW

File: .settings/org.eclipse.buildship.core.prefs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
+connection.project.dir=
+eclipse.preferences.version=1
+gradle.user.home=
+java.home=C\:/Program Files/Java/jdk1.8.0_241
+jvm.arguments=
+offline.mode=false
+override.workspace.settings=true
+show.console.view=true
+show.executions.view=true
```
Comment (NicholasFeldman) [inline .settings/org.eclipse.buildship.core.prefs:13]: What is this

Comment (rmw1075) [reply .settings/org.eclipse.buildship.core.prefs:13]: I have absolutely no idea. I think, when I opened the repository in VSCode to check something, it added it.

Comment (rmw1075) [reply .settings/org.eclipse.buildship.core.prefs:13]: Should I delete .settings/? it also looks like I can delete bin/

Comment (NicholasFeldman) [reply .settings/org.eclipse.buildship.core.prefs:13]: yes

Comment (rmw1075) [reply .settings/org.eclipse.buildship.core.prefs]: deleted

**Comment by rmw1075** [comment]
Merged /feature/dao into master

**Pull request merged**